### ISSUE TITLE
pacmap argument change.

### ISSUE THREAD

**Issue by ds-jim** [milescsmith/ReductionWrappers#19]
Hello, 

The pacmap wrapper needs to map `n_dims` to  `n_components` now.

**Comment by EDePasquale** [comment]
Yes, please fix this issue. When running pacmap I get this error: "Error in py_call_impl(callable, dots$args, dots$keywords) : 
TypeError: __init__() got an unexpected keyword argument 'n_dims'"
Thanks!
